RadixWallet/Features/DisplayEntitiesControlledByMnemonic/DisplayEntitiesControlledByMnemonic+View.swift

```diff
@@ -24,22 +28,30 @@ extension DisplayEntitiesControlledByMnemonic {
 		public var body: some SwiftUI.View {
 			WithViewStore(store, observe: { $0 }, send: { .view($0) }) { viewStore in
 				VStack(alignment: .leading) {
-					if viewStore.displayRevealMnemonicLink {
+					if viewStore.canNavigate {
```

Comment:
I would probably just have two different buttons, since the action and the label are both different. `if canReveal else if canRecover else...`

Reply:
sure, a bit of code duplication, but not worth handling...

Reply:
Or rather, it's a switch isn't it, with three cases.